ansible/archival-snapshots/resources/diff_script.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/env bash
+
+set -euxo pipefail
+
+FOREST=~/forest-v0.14.0/forest-tool
```

Comment:
You can't rely on this folder existing. If you need Forest, you have to download it yourself.

Reply:
fixed